testbed/datasenders/signalfx.go

```diff
@@ -65,7 +65,7 @@ func (sf *SFxMetricsDataSender) Start() error {
 	}
 
 	sf.Metrics = exporter
-	return nil
+	return exporter.Start(context.Background(), sf)
```

Comment:
Why is this necessary?

Reply:
It was not necessary, therefore removed it.